sqlitecluster/SQLite.cpp

```diff
@@ -73,8 +73,12 @@ SQLite::SQLite(const string& filename, int cacheSize, int autoCheckpoint, int ma
     // any tables to be effective.
     SASSERT(!SQuery(_db, "new file format for DESC indexes", "PRAGMA legacy_file_format = OFF"));
 
+    // Compare to see if "false" wasn't passed meaning the -synchronous flag was set and run the query
+    if (!SIEquals(synchronous, "false")) {
```

Comment:
Just do `if (!synchronous.empty())` and if the caller's specified a string, use that, otherwise, do nothing.